packages/mdc-tab-scroller/_mixins.scss

```diff
@@ -26,10 +26,28 @@
 @import "@material/feature-targeting/mixins";
 @import "./variables";
 
+@mixin mdc-tab-scroller-transition(
```

Comment:
Please add SassDoc (http://sassdoc.com/annotations/) for this mixin.

See https://github.com/material-components/material-components-web/blob/develop/packages/mdc-button/_mixins.scss#L237 for example.

You would also need to update README file in this folder.